tests/test/check/data/main.fmf

```diff
@@ -61,6 +61,32 @@
                       ls -alZ /root/passwd.log; \
                       rm -f /root/passwd.log" || /bin/true
 
+/coredump:
+    check:
+      - how: coredump
+    tag+:
+      - provision-only
+      - provision-container
+      - provision-virtual
```

Comment:
This tags are only used for the top-level test selection, not used under the `data` directories.

Reply:
Thanks!